infra/recipes/docker-compose/common/oauth/validate_token.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Check if token is provided
+if [ -z "$1" ]; then
+    echo "Error: Token not provided."
+    echo "Usage: $0 <token>"
+    exit 1
+fi
+
+KEYCLOAK_URL="http://localhost:8085"
+REALM="openhouse"
+TOKEN="$1"
+
+USERNAME=$(curl -X GET \
+  -H "Content-Type: application/x-www-form-urlencoded" \
+  -H "Authorization: Bearer $TOKEN" \
+  "$KEYCLOAK_URL/realms/$REALM/protocol/openid-connect/userinfo" | jq -r '.preferred_username')
```

Comment:
Nice, so this get the username from the local keycloak container.

Reply:
Yes, we pass the token to keycloak container, and it extracts the username for us.